source/_components/fan.knx.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "KNX Fan"
+description: "Instructions on how to integrate KNX fans with Home Assistant."
+date: 2018-09-05 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: knx.png
+ha_category: Fan
+ha_release: 0.78.0
+ha_iot_class: "Local Polling"
+---
+
+
+The `knx` fan component is used as in interface to fan actuators.
+
+The `knx` component must be configured correctly, see [KNX Component](/components/knx).
+
+## {% linkable_title Configuration %}
+
+To use your KNX fan in your installation, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+fan:
+  - platform: knx
+    name: living-room-fan
```

Comment:
The example may only contain the required variables to make it work, so remove `name` and `speed_state_address`